src/components/isearch/index.js

```diff
@@ -20,11 +20,40 @@ class ISearch extends Component {
       endScroll: false
     };
     this.handleResize = this.handleResize.bind(this);
+    this.loadQueryParams = this.loadQueryParams.bind(this);
   }
 
   componentWillMount () {
     window.addEventListener('resize', this.handleResize);
     this.addAnalyticsData();
+    let params = document.location.search.replace('?', '').split('&');
+    this.loadQueryParams(params);
+  }
+  loadQueryParams (params) {
+    if (!params.length) return;
+    const departureOptions = {
+      BLL: 'Billund - BLL',
+      CPH: 'København - CPH',
+      ALL: 'Aalborg - AAL',
+      ODE: 'Odense - ODE',
+      RNN: 'Rønne - RNN'
+    };
```

Comment:
This feels like something that should be extracted out into constants rather than being hidden in the code for a component.